builds/checkin/api-proxy.yaml

```diff
@@ -3,7 +3,14 @@ pr:
   branches:
     include:
       - master
-      - iiot
+      - release/*
+  paths:
+    include:
+      - "edge-modules/api-proxy-module/*"
```

Comment:
Looking at this a little more carefully....I don't think we want to include a `paths` element to the `pr` spec here, because the `check_run_pipeline` job is already doing that work.

(`paths` is a much simpler, cleaner way to restrict when this pipleline runs, but we added the `check_run_pipeline` job to our CI builds as a workaround for some problem related to kodiak--I don't remember the details.)

Reply:
Correct, having path filter does not work, I tried that before and failed :) The problem is that we have several required checks on PRs in GH, and if a required pipeline does not start (and it will not start if path filter does not match), no status is reported back to GH from ADO, and the PR is forever stuck in "waiting for pipeline" status